options.go

```diff
@@ -315,6 +351,19 @@ func parseSignatureKey(o *Options, msgs []string) []string {
 	return msgs
 }
 
+func parseJwtIssuers(o *Options, msgs []string) []string {
+	for _, jwtVerifier := range o.ExtraJwtIssuers {
+		components := strings.Split(jwtVerifier, "=")
+		if len(components) < 2 {
+			return append(msgs, "invalid jwt verifier uri=audience spec: "+
+				jwtVerifier)
+		}
+		uri, audience := components[0], strings.Join(components[1:], "=")
+		o.jwtIssuers = append(o.jwtIssuers, []string{uri, audience})
```

Comment:
I think we should make a small internal struct for this rather than just relying on a slice and the indexes of the slice, will make things neater, WDYT?

Reply:
Implemented a struct above, `JwtIssuer`. Should it be `jwtIssuer` or is it fine if it's exported?